api/src/test/java/org/openmrs/api/UserServiceTest.java

```diff
@@ -11,13 +11,12 @@
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNotSame;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
```

Comment:
You seem to be going back to JUnit 4. Was that intentional?

Reply:
yes is to correct this Error `UserServiceTest.getAllUsers_shouldNotContainsAnyDuplicateUsers:680 expected: <11> but was: <12>`  
for now

Reply:
Do you mean that we get this error on JUnit 5 but not JUnit 4?

Reply:
please remove this file from the PR. tests for it on JUnit 5 are green on Bamboo and the Travis CI build on master https://travis-ci.org/github/openmrs/openmrs-core/builds/708421471

Reply:
also see my suggestions you added some lines that have to be removed.